src/components/Onboarding/Screens/ReviewInformation.js

```diff
@@ -19,4 +63,544 @@ function ReviewInformation({ title }) {
+    <div
+      css={`
+        padding: ${5 * GU}px ${7 * GU}px 0px ${7 * GU}px;
+        width: 100%;
+      `}
+    >
+      <div>
+        <Field label="Garden name" value={config.garden.name} />
+        <Field label="Garden description" value={config.garden.description} />
+        <Field label="Forum" value={config.garden.forum} />
+      </div>
+      <LineBreak />
+      <div>
+        <div
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          ASSETS
+        </div>
+        <div
+          css={`
+            display: grid;
+            grid-template-columns: 1fr 1fr 1fr;
+            grid-gap: ${9 * GU}px;
+          `}
+        >
+          <Field
+            label="Header logo"
+            value={config.garden.logo_type?.blob.name || 'No image'}
+          />
+          <Field
+            label="Garden logo"
+            value={config.garden.logo?.blob.name || 'No image'}
+          />
+
+          <Field
+            label="Token icon"
+            value={config.garden.token_logo?.blob.name || 'No image'}
+          />
+        </div>
+        <p
+          css={`
+            ${textStyle('body4')};
+            color: ${theme.contentSecondary};
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          If you don´t upload images, default ones will be displayed.
+        </p>
+      </div>
+
+      <LineBreak />
+      <div>
+        <div
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          COMMUNITY LINKS
+        </div>
+        <div>
+          {Object.values(config.garden.links.community[0]).length > 0 ? (
+            config.garden.links.community.map(({ link, label }) => (
+              <Field label={label} value={link} />
+            ))
+          ) : (
+            <div
+              css={`
+                margin-bottom: ${3 * GU}px;
+              `}
+            >
+              No links
+            </div>
+          )}
+        </div>
+      </div>
+
+      <LineBreak />
+      <div>
+        <div
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        >
+          DOCUMENTATION LINKS
+        </div>
+        <div>
+          {Object.values(config.garden.links.documentation[0]).length > 0 ? (
+            config.garden.links.documentation.map(({ link, label }) => (
+              <Field label={label} value={link} />
+            ))
+          ) : (
+            <div
+              css={`
+                margin-bottom: ${3 * GU}px;
+              `}
+            >
+              No links
+            </div>
+          )}
+        </div>
+      </div>
+    </div>
+  )
+}
+/// /////// TOKENOMICS //////////
+function ReviewGardenTokenomics() {
+  return (
+    <div
+      css={`
+        padding: ${5 * GU}px ${7 * GU}px;
+        width: 100%;
+      `}
+    >
+      <ReviewTokens />
+      <LineBreak />
+      <ReviewHoneyswapLiquidity />
+      <LineBreak />
+      <ReviewIssuance />
```

Comment:
We need to check if we are on a Pre-existing token Garden then do not show this.